### PR TITLE
Dont show the git ref and context dir in the UI just include it in the actual link

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1364950

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

File: app/scripts/filters/util.js [hunk 1]
```diff
@@ -215,6 +215,28 @@ angular.module('openshiftConsole')
...
+    // Uses same regular expression for link detection as the angular linky filter, but just says whether something is a link or not
+    // adds start and end of line to the regexp to test if the entire string is a link
+    // See https://github.com/angular/angular.js/blob/master/src/ngSanitize/filter/linky.js
+    var LINKY_URL_REGEXP =
+        /^((ftp|https?):\/\/|(www\.)|(mailto:)?[A-Za-z0-9._%+-]+@)\S*[^\s.;,(){}<>"\u201d\u2019]$/i;
+    return function(link) {
+      if (!link) {
+        return link;
+      }
+      return LINKY_URL_REGEXP.test(link);
```
Comment (spadgett) [inline app/scripts/filters/util.js]: Does this work?

```js
var uri = new URI(link);
return uri.is('absolute');
```

Comment (jwforres) [inline app/scripts/filters/util.js]: That actually causes something like `git://git.eclipse.org/r/lyo/org.eclipse.lyo.ldp`   to be turned into a link

**Comment by jwforres** [comment]
updated with the isAbsoluteURL filter change

**Comment by spadgett** [comment]
[merge]

**Pull request merged**